src/package_tracking/src/package_edge_detection.cpp

```diff
@@ -0,0 +1,134 @@
+//
+// Created by Shahwaz Khan on 2/19/21.
+//
+
+#include "package_tracking/package_edge_detection.h"
+
```

Comment:
Add class level document on the purpose of the this class.

Reply:
Sorry bro, I don't understand what you are toying to say?..